public/sandbox-data/apprenticeship-levy/employers/222hmrctest4/employed/AA456789C.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": 200,
+  "jsonBody": {
+    "empref": "222/hmrctest4",
+    "nino": "AA456789C",
```

Comment:
Nino

Reply:
Ill address this in a later ticket. This is a bigger piece of work as App-levy is using itself as a stub. We need to create a separate microservice for this.